### PR TITLE
Add code and instructions to run quickstart on Colab.

### PR DESCRIPTION
**Pull request by fa9r**
## Describe changes
I added code and instructions to run quickstart on Colab.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Review by strickvl — APPROVED** [review]
Looks good to me!

File: examples/quickstart/notebooks/quickstart.ipynb [hunk 1]
```diff
@@ -767,7 +800,7 @@
    "provenance": []
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.8.13 64-bit ('temp')",
```
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:803]:
```suggestion
   "display_name": "Python 3",
```

File: examples/quickstart/notebooks/quickstart.ipynb [hunk 2]
```diff
@@ -146,7 +148,12 @@
    "source": [
     "## Using Google Colab\n",
     "\n",
-    "If you follow this quickstart in Google's Colab, you will need an [ngrok account](https://dashboard.ngrok.com/signup) to view some of the visualizations later. Please set up an account, then set your user token below:"
+    "If you follow this quickstart in Google's Colab, make sure you successfully\n",
+    "installed ZenML in the previous code cell.\n",
+    "\n",
+    "Furthermore, you will need an [ngrok account](https://dashboard.ngrok.com/signup) \n",
+    "to view some of the visualizations later. \n",
+    "Please set up an account, then set your user token below:"
```
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:156]:
```suggestion
    "If you follow this quickstart in Google's Colab, you will need an [ngrok account](https://dashboard.ngrok.com/signup) to view some of the visualizations later. Please set up an account, then set your user token below:"
```

File: examples/quickstart/notebooks/quickstart.ipynb [hunk 3]
```diff
@@ -113,7 +113,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, for the quickstart, we need install some dependencies. Once you have ZenML installed, you can use our CLI to install the required integrations."
+    "Now, for the quickstart, we need to install some dependencies. Once you have ZenML installed, you can use our CLI to install the required integrations.\n",
+    "\n",
+    "**Colab Note**: If you are running this notebook on Colab, make sure to uncomment the `pip install zenml` command below!"
```
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:118]:
```suggestion
```

File: examples/quickstart/notebooks/quickstart.ipynb [hunk 4]
```diff
@@ -122,7 +124,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# %pip install \"zenml[server]\"  # install ZenML\n",
+    "# %pip install \"zenml[server]\"  # install ZenML - uncomment this on Colab\n",
```
Comment (fa9r) [inline examples/quickstart/notebooks/quickstart.ipynb:127]:
```suggestion
    "%pip install \"zenml[server]\"  # install ZenML\n",
```

**Review by fa9r — COMMENTED** [review]
Cleaner to uncomment the pip install directly instead of adding all those comments.

**Pull request merged**